### DISCUSSION TITLE
COMEX STAT > Importação e Exportação

### DISCUSSION THREAD

**Discussion by reinaldodelfiaco** [BrasilAPI/BrasilAPI#219] [Ideas]
Relação mensal dos produtos exportados e importados por ano, mês, código NCM, código da unidade estatística, código de país de destino/origem do produto, código da UF de origem/destino do produto, código da via de transporte, código da URF de embarque/desembarque, quantidade estatística, quilograma líquido, valor dólar FOB (US$).

https://www.gov.br/produtividade-e-comercio-exterior/pt-br/assuntos/comercio-exterior/estatisticas/base-de-dados-bruta

Útil em regressões econométricas, pesquisas de comércio exterior e notas fiscais.

**Comment by mairacanal** [comment]
Talvez poderia ser algo como o Observatory of Economic Complexity (https://oec.world/). Inclusive, eles estão desenvolvendo uma API, atualmente, para obter dados econômicos de todo o mundo (https://legacy.oec.world/api/).

**Comment by lucianopf** [comment]
@reinaldodelfiaco consegue descrever um pouco mais sobre a ideia? 

Acho que tem coisa interessante nessa proposta mas acredito que me falta contexto pra conseguir captar totalmente a ideia 😬
